ios/RNWrappers.m

```diff
@@ -0,0 +1,235 @@
+// MARK: - Public methods
+//---------------------------------------------------
+
++ (JudoKit *)judoSessionFromProperties:(NSDictionary *)properties {
+  NSString *token = [RCTConvert NSString:properties[@"token"]];
+  NSString *secret = [RCTConvert NSString:properties[@"secret"]];
+  BOOL isSandboxed = [RCTConvert BOOL:properties[@"sandboxed"]];
+
+  JudoKit *judoKit = [[JudoKit alloc] initWithToken:token secret:secret];
+  judoKit.isSandboxed = isSandboxed;
+
+  return judoKit;
+}
+
++ (TransactionType)transactionTypeFromProperties:(NSDictionary *)properties {
+
+    int intType = [RCTConvert int:properties[@"transactionType"]];
+
+    NSArray<NSNumber *> *availableTypes = @[
+        @(TransactionTypePayment),
+        @(TransactionTypePreAuth),
+        @(TransactionTypeRegisterCard),
+        @(TransactionTypeCheckCard),
+        @(TransactionTypeSaveCard)
+    ];
+
+    return availableTypes[intType].intValue;
+}
+
++ (TransactionMode)transactionModeFromProperties:(NSDictionary *)properties {
+    int intType = [RCTConvert int:properties[@"transactionMode"]];
+    return intType == 0 ? TransactionModePayment : TransactionModePreAuth;
+}
+
++ (JPConfiguration *)configurationFromProperties:(NSDictionary *)properties {
+
+  NSDictionary *configurationDict = properties[@"configuration"];
+
+  NSString *judoId = [RCTConvert NSString:configurationDict[@"judoId"]];
+  JPAmount *amount = [RNWrappers amountFromConfiguration:configurationDict];
+  JPReference *reference = [RNWrappers referenceFromConfiguration:configurationDict];
+
+  JPConfiguration *configuration = [[JPConfiguration alloc] initWithJudoID:judoId
+                                                                    amount:amount
+                                                                 reference:reference];
+
+  configuration.siteId = [RCTConvert NSString:configurationDict[@"siteId"]];
+  configuration.uiConfiguration = [RNWrappers uiConfigurationFromConfiguration:configurationDict];
+  configuration.supportedCardNetworks = [RNWrappers cardNetworksFromConfiguration:configurationDict];
+  configuration.primaryAccountDetails = [RNWrappers accountDetailsFromConfiguration:configurationDict];
+  configuration.cardAddress = [RNWrappers cardAddressFromConfiguration:configurationDict];
+  configuration.paymentMethods = [RNWrappers paymentMethodsFromConfiguration:configurationDict];
+  configuration.applePayConfiguration = [RNApplePayWrappers applePayConfigurationFromConfiguration:configurationDict];
+
+  return configuration;
+}
+
++ (CardNetwork)cardNetworksFromConfiguration:(NSDictionary *)configuration {
+
+    CardNetwork supportedCardNetworks = CardNetworkUnknown;
+
+    int supportedNetworksBitmask = [RCTConvert int:configuration[@"supportedCardNetworks"]];
+
+    if (supportedNetworksBitmask & IOSCardNetworkAll) {
+        return CardNetworksAll;
+    }
+
+    if (supportedNetworksBitmask & IOSCardNetworkVisa) {
+        RCTLog(@"ADDING VISA");
```

Comment:
do we need `RCTLog's` in this file?

Reply:
No longer present since merging last branch